core/Updates/4.0.0-b1.php

```diff
@@ -121,6 +142,11 @@ public function doUpdate(Updater $updater)
             // switch to default provider if GeoIp Legacy was still in use
             LocationProvider::setCurrentProvider(LocationProvider\DefaultProvider::ID);
         }
+
+        // update currently used database charset in config
+        $config = Config::getInstance();
```

Comment:
BTW... I wonder if it be easy enough to do similar to DB and plugin whether we could add a 

```php
$this->migration->config->changeConfigValue('database', 'charset', DbHelper::getDefaultCharset());
```

This command would then print a command like `./config:set --section=database --key=charset --value=XYZ`
in the list of commands to execute just to make sure this one isn't missed. I suppose if this wasn't executed for some reason otherwise it could cause issues?

Reply:
makes sense... will add that